BDLeu9

thea-m opened the pull request:
new scroll :) enjoy!

MassimoVilla commented on OxfordBodleian/Ullendorff/BDLeu9.xml:
```diff
@@ -34,9 +37,81 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                            <textLang mainLang="gez"/>
+                            <!-- this is the entire text-->
+                            <incipit xml:lang="gez">
+                                <hi rend="rubric"><sic>ቀ</sic> በስመ፡ <add place="interlinear">ወ</add>ወልድ፡ ወመንፈ</hi>ስ፡ ቅዱስ፡
```
Since this is a well-established formula, I would add `<supplied reason="omitted">አብ፡</supplied>` after በስመ፡

MassimoVilla reviewed (changes requested):
Thank you Thea. Just one minor remark.

ElianaDalSasso commented on OxfordBodleian/Ullendorff/BDLeu9.xml:
```diff
@@ -45,48 +121,132 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                        <bindingDesc>
+                            <binding>
+                                <decoNote xml:id="b1">The two parchment strips that compose the scroll are sewn together with a running stitch.</decoNote>
+                                <decoNote xml:id="b2">The manuscript is kept in a pipe case.</decoNote>
```
So cute! :3

ElianaDalSasso reviewed (changes requested):
Thank you Thea! I have just two minor comments. One regards the presence of a tear next to the hole in the upper part, the second regards the dating since 2002 and 2006 appear.

thea-m commented on OxfordBodleian/Ullendorff/BDLeu9.xml:
```diff
@@ -45,48 +121,132 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                           
+                           <desc>Modern bulky script, somewhat irregular. The nine-dotted asterisk is used often as word separator, and the word separator is often omitted when the end of a word 
+                               coincides with the end of the line.</desc>
+                                <date notBefore="1800" notAfter="2006"/>
```
It was because the price is given in francs, but you are right, I didn't explain this. I've modified the formulation, and updated the date in palaeography.
On second thoughts, theoretically Swiss francs could also be meant...

thea-m commented:
Thank you, I incorporated the corrections and also added the edition, which I'd previously forgotten (irrelevant for the printed catalogue, just for consistency)

ElianaDalSasso reviewed (approved):
Thanks Thea! Now it is very clear!
